packages/core/src/node/messaging/ipc-connection-provider.ts

```diff
@@ -81,11 +81,11 @@ export class IPCConnectionProvider {
             error: (message: string) => this.logger.error(`[${options.serverName}: ${childProcess.pid}] ${message}`),
             warn: (message: string) => this.logger.warn(`[${options.serverName}: ${childProcess.pid}] ${message}`),
             info: (message: string) => this.logger.info(`[${options.serverName}: ${childProcess.pid}] ${message}`),
-            log: (message: string) => this.logger.info(`[${options.serverName}: ${childProcess.pid}] ${message}`)
+            log: (message: string) => this.logger.debug(`[${options.serverName}: ${childProcess.pid}] ${message}`)
         });
-        connection.trace(Trace.Off, {
+        connection.trace(Trace.Verbose, {
```

Comment:
@alvsan09 I believe the idea would be to make this value configurable, not hardcoded to `off` or `verbose`.